City-of-Helsinki/helsinki-paatokset
# PP-283: Fixed accessibility issues and initiatives listing

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-283

Made accessibility fixes mentioned in the ticket and fixed issues in initiatives listing

**How to test**

Checkout branch and run `drush cim -y; drush cr`

Go to https://helsinki-paatokset.docker.so/fi/kokouskalenteri 

- check that the links have now an aria-label
- make sure that the calendar arrows are accessible with the keyboard and enter is registered as a click.
- the date headers should now be h3 and meeting titles h4

Go to some policymaker page  i.ex. https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginvaltuusto

- labels should now be binded to inputs 
- member name is now a h4-tag
- there is a text when there are no search results


Go to https://helsinki-paatokset.docker.so/fi/aloitteet or create a page with this path

- This listing should now work properly and  organised in date order 
- it should also now render to pages with paths /aloitteet, /motioner /motions
